ttnn/cpp/ttnn/operations/ccl/all_gather/all_gather.hpp

```diff
@@ -5,19 +5,25 @@
 #pragma once
 
 #include "ttnn/decorators.hpp"
+#include "ttnn/operations/ccl/ccl_host_datastructures.hpp"
 
 namespace ttnn {
 namespace operations {
 namespace ccl {
 
+namespace all_gather_op {
+using ttnn::ccl::Topology;
```

Comment:
I'm confused why this was namespaced to `all_gather_op`?

Reply:
I have done this as the same method is followed in `all_gather_op.hpp`, I will revert this change to call directly without adding additional namespace